internal/gapicgen/generator/gapics.go

```diff
@@ -18,11 +18,14 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io/fs"
```

Comment:
We should update the go version in the go.mod to 1.16 if this package is now being used.

Reply:
Done.